supplements/fizbans-treasury-of-dragons/spells.xml

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
+	<info>
+		<name>Spells</name>
+		<description>The Spells from Fizban's Treasury Of Dragons</description>
+		<author url="https://dnd.wizards.com/products/treasury-dragons">Fizban's Treasury Of Dragons</author>
+		<update version="0.0.1">
+			<file name="spells.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/supplements/fizbans-treasury-of-dragons/spells.xml" />
+		</update>
+	</info>
+	
+	<element name="Ashardalon's Stride" type="Spell" source="Fizban's Treasury Of Dragons" id="ID_WOTC_FTOD_SPELL_ASHARDALONS_STRIDE">
+		<supports>Artificer, Ranger, Sorcerer, Wizard</supports>
+		<description>
+			<p>The billowing flames of a dragon blast from your feet, granting you explosive speed. For the duration, your speed increases by 20 feet and moving doesn’t provoke opportunity attacks.</p>
+			<p class="indent">When you move within 5 feet of a creature or an object that isn’t being worn or carried, it takes 1d6 fire damage from your trail of heat. A creature or object can take this damage only once during a turn.</p>
+			<p class="indent"><strong><em>At Higher Levels.</em></strong> When you cast this spell using a spell slot of 4th level or higher, increase your speed by 5 feet for each spell slot level above 3rd. The spell deals an additional 1d6 ¦re damage for each slot level above 3rd.</p>
+		</description>
+		<setters>
+			<set name="level">3</set>
+			<set name="school">Transmutation</set>
+			<set name="time">1 bonus action</set>
+			<set name="range">Self</set>
+			<set name="hasVerbalComponent">true</set>
+			<set name="hasSomaticComponent">true</set>
+			<set name="hasMaterialComponent">false</set>
+			<set name="duration">Concentration, up to 1 minute</set>
+			<set name="isConcentration">true</set>
+		</setters>
```

Comment:
I'm a little confused why none of the spells have the isRitual setter

Reply:
Spells in TCOE don't either, if they aren't concentration they don't have isConcentration there either, I assume they're automatically set to false (which I just missed when making the file, so could remove those)